src/Api/Command/PayByBlikHandler.php

```diff
@@ -29,13 +28,4 @@ private function setTransactionData(PaymentInterface $payment, string $blikToken
 
         $payment->setDetails($paymentDetails->toArray());
     }
-
-    private function createResultFrom(PaymentInterface $payment): PayResult
```

Comment:
FYI I moved these functions to Abstract (one that Handlers extends from) as in most part they were duplicated